src/node/gov/handlers/proposals.h

```diff
@@ -131,10 +133,14 @@ namespace ccf::gov::endpoints
       ccf::jsgov::ProposalInfo& proposal_info,
       const std::string& constitution)
     {
-      std::vector<std::pair<MemberId, bool>> votes;
-      std::optional<jsgov::Failure> failure = std::nullopt;
-      std::optional<ccf::jsgov::Votes> final_votes = std::nullopt;
-      std::optional<ccf::jsgov::VoteFailures> vote_failures = std::nullopt;
+      // Create some temporaries to store resolution progress. These are written
+      // to proposal_info, and the KV, only for completed (accepted) proposal.
```

Comment:
Surely they should be written for failed/rejected proposals too?

Reply:
It looks like they are, the comment just needs updating.

Reply:
Yes - they are, and the comment is now updated.